index.html

```diff
@@ -350,66 +350,93 @@ <h3>Use Cases and Requirements</h3>
 specifically:
         </p>
 
-        <ul>
+        <ol>
           <li>
-<a>Holders</a> receive and store <a>verifiable credentials</a> from
-<a>issuers</a>. <a>Holders</a> might interact with an <a>issuer</a> through an
-agent that the <a>issuer</a> does not need to trust.
+<a>Verifiable Credentials</a> represent statements made by an <a>Issuer</a> in
```

Comment:
Have you mapped all of the old requirements to the new requirements for the ecosystem? I can't easily tell if we're losing capabilities w/ the new list that we were asserting in the older list. The reason I'm concerned about this is that it took us 6+ months to come up with the old list, so I want to make sure we're not losing capability expectations.

Reply:
We were careful and methodical, starting from the original list. I think the biggest change was dropping the specific term “derived predicate” to introduce selective disclosure language because “derived predicate” read like an implementation detail of a specific ZKP rather than a market requirement. (You called that out below). As expected, you have several comments that we’ll need to consider to make sure we’re all aligned.

Reply:
Ok, makes me feel better about the rework that it was done in a methodical fashion, I'll most likely defer to each of you unless I see something glaring.

wrt. "derived predicate" -- agree that the language in the spec needs to change, it sticks out as a sore thumb. If we keep it, I suggest changing it to something like "derived attribute" or something of that nature.